src/iotHubResourceExplorer.ts

```diff
@@ -90,7 +90,7 @@ export class IoTHubResourceExplorer extends BaseExplorer {
             }, 1000);
 
             const { session, subscription } = subscriptionItem;
-            const client = new IotHubClient(session.credentials, subscription.subscriptionId);
+            const client = new IotHubClient(session.credentials, subscription.subscriptionId, session.environment.resourceManagerEndpointUrl);

```

Comment:
check whether it is manager endpoint or resource manage endpoint?

Reply:
Thanks. I have double check it should be `resourceManagerEndpointUrl`